CONTRIBUTING.md

```diff
@@ -238,6 +238,19 @@ need to "save" the changes.
    - This will modify the appropriate patches based on your commits.
 1. PR your modified patch file(s) back to this repository.
 
+## Rebasing PRs
+Steps to rebase a pr to include the latest changes from `master`.
+
+1. Pull latest changes from upstream's master; `git checkout master && git pull`.
+2. Checkout feature/fix branch and rebase on master; `git checkout patch-branch && git rebase master`.
+3. Apply updated patchs; `./paper p`.
+4. If there are conflicts, fix them.
+5. If your PR creates new patches (not just modifies existing ones), in both the `Paper-Server` and `Paper-API` directories:
+    1. Ensure your newly-created patch is the last commit.
```

Comment:
```suggestion
5. If your PR creates new patches instead of modifying existing ones, in both the `Paper-Server` and `Paper-API` directories ensure that:
    1. Your newly-created patch is the last commit.
```